### PR TITLE
Renders alt text for media

### PR DESCRIPTION
**Pull request by UsmanMuhammad**
REDMINE-17864

It renders alt texts if provided or empty
text for all background and inline images,
videos, and audios

beforeAfter images already had a mechanism
to render alt texts

We already get the files in video, and audio player.
It sends the value to MediaPool where it adds the attribute
to the player

### CODE REVIEW

File: package/src/frontend/media/MediaPool.js [hunk 1]
```diff
@@ -44,7 +44,8 @@ export class MediaPool {
 
     player = this.unAllocatedPlayers[playerType].pop();
     if (player) {
-      
+
+      player.el().children[0].setAttribute('alt', altText); //add alt texts;
```
Comment (tf) [inline package/src/frontend/media/MediaPool.js:48]: **suggestion**: Use `getMediaElement`

Two lines down we also change attributes on the media element. By using `getMediaElement`, we get around duplicating the knowledge that VideoJS wraps the media element in a div. The comment does not add any value.

File: entry_types/scrolled/package/spec/frontend/AudioPlayer-spec.js [hunk 2]
```diff
@@ -136,4 +136,45 @@ describe('AudioPlayer', () => {
...
+
+  it('renders empty alt attr', () => {
+    const result =
+      renderInEntry(<AudioPlayer {...requiredProps()} id={100} />, {
+        seed: getAudioFileSeed({
+          permaId: 100
+        })
+      });
+
+    expect(result.container.querySelector('audio')).toHaveAttribute('alt', ' ');
```
Comment (tf) [inline entry_types/scrolled/package/spec/frontend/AudioPlayer-spec.js:178]: **question**: Why do we default to a space character?

File: package/src/frontend/media/MediaPool.js (hunk 1 above)
Comment (tf) [reply package/src/frontend/media/MediaPool.js:48]: **suggestion**: Add test

This is covered indirectly through the React component tests. Those components belong to a different package (`pageflow-scrolled`), though. We could also add a test in the `pageflow` package. Similar to [the one for loop](https://github.com/codevise/pageflow/blob/93180fde9abe03ae24b7aff37969083a44bd3b96/package/spec/frontend/media/media_spec.js#L44)

**Review by tf — CHANGES_REQUESTED** [review]

File: entry_types/scrolled/package/spec/frontend/AudioPlayer-spec.js (hunk 2 above)
Comment (UsmanMuhammad) [reply entry_types/scrolled/package/spec/frontend/AudioPlayer-spec.js:178]: Removed that.

File: package/src/frontend/media/MediaPool.js (hunk 1 above)
Comment (UsmanMuhammad) [reply package/src/frontend/media/MediaPool.js:48]: Done

**Review by tf — APPROVED** [review]

**Pull request merged**